.github/workflows/update-prerun-script.yml

```diff
@@ -0,0 +1,45 @@
+name: Update pre build script on Saucelabs
+
+on:
+  schedule:
+    - cron: '0 0 1 */2 *'
+
+env:
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  download-pre-run-script:
+    name: Download pre run script from S3 bucket
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - run: aws s3 cp s3://saucelabs-prebuild-script/script.sh ./
```

Comment:
what is the point of downloading this in a separate job? Each job runs in a different context, right?